### PR TITLE
chore: Modify button token annotation error

### PR DESCRIPTION
**Pull request by sun-shu**
fix: Modify button token annotation error

[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [X] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 💡 Background and solution
Modify button token annotation error 
修正文档中关于按钮token变量中文描述错误, paddingBlock是纵向，写成横向了。         

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |Modify button token annotation error  |
| 🇨🇳 Chinese | 修正文档中关于按钮token变量中文描述错误 |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [X] Doc is updated/provided or not needed
- [X] Demo is updated/provided or not needed
- [X] TypeScript definition is updated/provided or not needed
- [X] Changelog is provided or not needed

### CODE REVIEW

File: components/button/style/index.ts [hunk 1]
```diff
@@ -91,17 +91,17 @@ export interface ComponentToken {
    */
   paddingInlineSM: CSSProperties['paddingInline'];
   /**
-   * @desc 按钮横向内间距
+   * @desc 按钮纵向内间距
    * @descEN Horizontal padding of button
    */
   paddingBlock: CSSProperties['paddingInline'];
```
Comment (afc163) [inline components/button/style/index.ts:97]: 这里比较奇怪，前面是 paddingBlock 后面是 paddingInline，英文是 Horizontal，中文是 纵向

Comment (afc163) [reply components/button/style/index.ts:97]: 哪个是对的？

Comment (afc163) [reply components/button/style/index.ts:97]: ping

Comment (MadCcc) [reply components/button/style/index.ts:97]: 应该是纵向

Comment (afc163) [inline components/button/style/index.ts:97]:
```suggestion
  paddingBlock: CSSProperties['paddingBlock'];
```

File: components/button/style/index.ts [hunk 2]
```diff
@@ -91,17 +91,17 @@ export interface ComponentToken {
    */
   paddingInlineSM: CSSProperties['paddingInline'];
   /**
-   * @desc 按钮横向内间距
+   * @desc 按钮纵向内间距
    * @descEN Horizontal padding of button
```
Comment (afc163) [inline components/button/style/index.ts:95]:
```suggestion
   * @descEN Vertical padding of button
```

File: components/button/style/index.ts [hunk 3]
```diff
@@ -91,17 +91,17 @@ export interface ComponentToken {
...
   /**
-   * @desc 大号按钮横向内间距
+   * @desc 大号按钮纵向内间距
    * @descEN Horizontal padding of large button
    */
   paddingBlockLG: CSSProperties['paddingInline'];
   /**
-   * @desc 小号按钮横向内间距
+   * @desc 小号按钮纵向内间距
    * @descEN Horizontal padding of small button
```
Comment (afc163) [inline components/button/style/index.ts:105]:
```suggestion
   * @descEN Vertical padding of small button
```

File: components/button/style/index.ts [hunk 4]
```diff
@@ -91,17 +91,17 @@ export interface ComponentToken {
...
-   * @descEN Horizontal padding of button
+   * @desc 按钮纵向内间距
+   * @descEN Vertical padding of button
    */
-  paddingBlock: CSSProperties['paddingInline'];
+  paddingBlock: CSSProperties['paddingBlock'];
   /**
-   * @desc 大号按钮横向内间距
+   * @desc 大号按钮纵向内间距
    * @descEN Horizontal padding of large button
```
Comment (afc163) [inline components/button/style/index.ts:100]:
```suggestion
   * @descEN Vertical padding of large button
```

File: components/button/style/index.ts [hunk 5]
```diff
@@ -91,17 +91,17 @@ export interface ComponentToken {
...
    */
   paddingBlockLG: CSSProperties['paddingInline'];
```
Comment (afc163) [inline components/button/style/index.ts:102]:
```suggestion
  paddingBlockLG: CSSProperties['paddingBlock'];
```

File: components/button/style/index.ts [hunk 6]
```diff
@@ -91,17 +91,17 @@ export interface ComponentToken {
...
   /**
-   * @desc 小号按钮横向内间距
+   * @desc 小号按钮纵向内间距
    * @descEN Horizontal padding of small button
    */
   paddingBlockSM: CSSProperties['paddingInline'];
```
Comment (afc163) [inline components/button/style/index.ts:107]:
```suggestion
  paddingBlockSM: CSSProperties['paddingBlock'];
```

**Review by afc163 — DISMISSED** [review]

**Review by afc163 — APPROVED** [review]

**Pull request merged**